lcls_tools/common/devices/wire.py

```diff
@@ -93,21 +95,21 @@ def validate_pv_fields(cls, v: str) -> PV:
 
 class WireControlInformation(ControlInformation):
     PVs: SerializeAsAny[WirePVSet]
-    _ctrl_options: SerializeAsAny[Optional[Dict[str, int]]] = dict()
+    # _ctrl_options: SerializeAsAny[Optional[Dict[str, int]]] = dict()
 
```

Comment:
Were all the control options removed bc they are in the PVs that Ziyu gave us?

Reply:
Yes, basically